### PR TITLE
inbox 1 screen

### PR DESCRIPTION
**Pull request by shlee2017**

### CODE REVIEW

File: front_end/Inbox/InboxOne.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    ScrollView,
+    Text,
+    TouchableHighlight,
+    View,
+    SafeAreaView
+} from 'react-native';
+
+import Message from './Message';
+import Icon from '../Utility/Icon';
+import styles from '../Static/style';
```
Comment (xumaple) [inline front_end/Inbox/InboxOne.js:13]: nit: maybe rename this to main_style? dunno. Can be a separate PR too

File: front_end/Inbox/Message.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+            name: props.name,
+            message: props.message,
+            time: props.time,
+            chatID: props.chatID
+            //navigation: props.navigation for later
+        };
+        this.onPressMessage = this.onPressMessage.bind(this);
+    }
+
+    onPressMessage() {
```
Comment (xumaple) [inline front_end/Inbox/Message.js:25]: What is this function supposed to do??

Comment (shlee2017) [reply front_end/Inbox/Message.js:25]: it is suppose to go to the chat room. I just put a console log just temporarily
